spec/specification/dsl_spec.rb

```diff
@@ -368,6 +368,27 @@ module Pod
 
     #-----------------------------------------------------------------------------#
 
+    describe 'Test specs' do
+      before do
+        @spec = Spec.new do |spec|
+          spec.name = 'Spec'
+          spec.test_spec do |test_spec|
+            test_spec.test_type = :unit
```

Comment:
@orta @DanToml if I a podspec creator does not call this, how can I enforce a default value? Is it on the `Specification` constructor that I add `test_spec = :unit if test_specification` or something akin to that?